example/docker-compose.yml

```diff
@@ -1,38 +1,41 @@
-version: '3'
+version: "3"
```

Comment:
formatted this file with prettier. that's the reason for the whitespace changes.